back/spec/fixtures/template_without_images.yml

```diff
@@ -844,8 +844,7 @@ models:
         information, ... After you''ve done that, start sharing your idea on social
         media or through email. Invite people to join the discussion on your idea,
         or simply convince them to support it by voting in favour.</p><p><a href="/en/projects/open-idea-project/ideas/new"
-        target="_blank"><img src="https://res.cloudinary.com/citizenlabco/image/upload/v1540212640/image_h6upkl.png"
-        data-align="center" style="display: block;margin:auto;"></a></p><p><strong>About
+        target="_blank"></a></p><p><strong>About
```

Comment:
Since the original images no longer exist on Cloudinary, and we have not had issues with this, plus the filename implies images should not be included, the simplest approach seems to be to just remove the image elements containing Cloudinary URLs from this file.